src/Code.js

```diff
@@ -58,27 +38,47 @@ function setApiKey() {
   const button = result.getSelectedButton();
   const text = result.getResponseText();
   if (button === ui.Button.OK) {
-    setProperty(PROP_MB_API_KEY, text);
-    ui.alert(`API key is ${text}.`);
+    try {
+      setProperty(PROP_MB_API_KEY, text);
+      ui.alert(`API key is ${text}`);
+    } catch (e) {
+      ui.alert(e.message);
+    }
   }
 }
 
 function deleteAllSettings() {
-  const properties = PropertiesService.getDocumentProperties();
-  if (properties) {
-    properties.deleteAllProperties();
-  } else {
-    testProperties = {};
+  const ui = SpreadsheetApp.getUi();
+  const result = ui.alert(
+    'Are you sure to delete all settings?',
+    ui.ButtonSet.YES_NO,
```

Comment:
I know it's already `deleteAllSettings()` from a previous PR, but let's change this to be `resetCredentials()` and `"Are you sure you want to reset the Deployment ID and API key?"`, since that's the only/primary use for this at the moment.